nextflow_schema.json

```diff
@@ -2,109 +2,130 @@
-                "blast_max_hsps": {
-                    "type": "integer",
-                    "default": 1
+                "skip_blastn": {
+                    "type": "boolean",
+                    "description": "Defines if blastn should be used or not. By default blastn is used in the workflow. If also the filter is enabled the reads are filtered using the output from blastn."
                 },
-                "blast_max_target_seqs": {
-                    "type": "integer",
-                    "default": 1
+                "filter_with_kraken2": {
+                    "type": "boolean",
+                    "description": "If this is set to True the kraken2 ouput is used for filtering."
                 }
             }
         },
-        "workflow_parameters": {
-            "title": "Workflow parameters",
+        "kraken2": {
+            "title": "kraken2",
             "type": "object",
-            "description": "",
+            "description": "Parameters used by kraken2 to classify all reads provided.",
             "default": "",
             "properties": {
-                "single_end": {
-                    "type": "boolean"
-                },
-                "long_read": {
-                    "type": "boolean"
-                },
-                "enable_filter": {
-                    "type": "boolean"
+                "kraken2db": {
+                    "type": "string",
+                    "default": "https://genome-idx.s3.amazonaws.com/kraken/k2_standard_08gb_20231009.tar.gz",
+                    "description": "The database which is used in the classification step. For input how to use this parameter to fine-tune the step refer to usage/Databases/kraken2"
                 },
-                "filter_trimmed": {
-                    "type": "boolean"
+                "save_output_fastqs": {
+                    "type": "boolean",
+                    "description": "If the unclassified and classified reads should be saved to different files. The default false should be used",
+                    "hidden": true
                 },
-                "skip_blastn": {
-                    "type": "boolean"
+                "save_reads_assignment": {
+                    "type": "boolean",
+                    "default": true,
+                    "hidden": true,
+                    "description": "If the read assignments should be saved. Leave this to be true"
                 },
-                "filter_with_kraken2": {
-                    "type": "boolean"
+                "kraken2confidence": {
+                    "type": "number",
+                    "default": 0.05,
+                    "description": "Confidence in the classification of a read as a certain taxon. Refere to https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown#confidence-scoring for details."
                 }
             }
         },
         "kraken2taxa": {
             "title": "kraken2taxa",
```

Comment:
That looks oddly placed in https://nf-co.re/detaxizer/dev/parameters
Why not below kraken2? Or under a more descriptive title?